### PR TITLE
feat(react-composer): plain text support

### PR DESCRIPTION
**Pull request by wittjosiah**

### CODE REVIEW

**Comment by wittjosiah** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2696** <a href="https://app.graphite.dev/github/pr/dxos/dxos/2696" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dxos/dxos/2696?utm_source=stack-comment).

File: packages/apps/composer-app/src/pages/DocumentPage.tsx [hunk 1]
```diff
@@ -38,33 +39,31 @@ const nestedParagraphOutput = / +\n/g;
 
 const PureDocumentPage = observer(({ document }: { document: ComposerDocument }) => {
   const { t } = useTranslation('composer');
-
-  const editor = useComposerEditor({
-    document: document.content,
-    slots: { editor: { className: 'pbe-20' } }
-  });
+  const editorRef = useRef<TipTapEditor>(null);
```
Comment (thure) [inline packages/apps/composer-app/src/pages/DocumentPage.tsx:42]: Ah I was wondering if I shoulda just used `ref`. Thanks for refactoring to this.

File: packages/experimental/kai/src/frames/Document/DocumentFrame.tsx [hunk 2]
```diff
@@ -57,8 +57,8 @@ export const DocumentFrame = observer(() => {
             }}
           />
 
-          <Composer
-            document={document.content}
+          <RichTextComposer
+            text={document.content}
```
Comment (thure) [inline packages/experimental/kai/src/frames/Document/DocumentFrame.tsx:61]: I like this better 👍

**Review by thure — APPROVED** [review]
lgtm ✨

**Pull request merged**